### DISCUSSION TITLE
simple-data-analysis.js library

### DISCUSSION THREAD

**Discussion by nshiab** [duckdb/duckdb-spatial#335] [Show and tell]
I just wanted to thank you for all of your work! I am a computational journalist and senior data producer for [CBC News](https://www.cbc.ca/news). I am using duckdb_spatial for our library [simple-data-analysis.js](https://github.com/nshiab/simple-data-analysis).

To produce data journalism projects, data crunching and analysis are usually done with Python or R, and interactive data visualizations are coded in JavaScript. However, being proficient in multiple programming languages is hard.

The missing piece in the JavaScript/TypeScript ecosystem was an easy-to-use and performant library for data analysis. This is why SDA was created, and it's powered by DuckDB and your extension!

Let me know what you think. And thanks again! :)

**Comment by Constantine234** [comment]
sorry to ask this but does this mean you can use JavaScript/TypeScript for analysis or part of analysis?
just curious

**Reply by nshiab** [reply]
I use it for analysis in our data-driven projects. You can have a look at the examples in the Readme: https://github.com/nshiab/simple-data-analysis

Let me know what you think. :)

**Reply by Constantine234** [reply]
alright bro thanks for this nice to meet you
